src/routes/loginOAuth.js

```diff
@@ -0,0 +1,43 @@
+var express = require("express");
+const Users = require("../models/users.js");
+
+var encrypt = require("../services/encryptPassword");
+
+const server = express();
+
+server.use(express.json());
+
+server.post("/auth/login", async (req, res) => {
+  const { username, password } = req.body;
+
+  const user = await Users.findOne({ username });
+  if (!user) {
+    res.status(401).json({
+      message: "Login not successful",
+      error: "401: User not found",
+    });
+  } else {
+    var encrPassword;
```

Comment:
Please use let here instead of var
```suggestion
    let encrPassword;
```